### ISSUE TITLE
Support more input types

### ISSUE THREAD

**Issue by daattali** [daattali/shinyforms#4]
- checkbox
- radio  
- select (dropdown)
- paragraph
- date
- slider?
- linear scale?

**Comment by fozy81** [comment]
Just a small change to add date and slide:

https://github.com/daattali/shinyforms/pull/15

**Comment by daattali** [comment]
Thank you, but it's really hard to tell what's going on in those diffs, the entire file almost is added and removed. Also, do you think sliders are necessary? That's why I put them in question mark.  They're not a very common input type.  Most forms have something like a "scale" of radio buttons, but I feel like sliders are very big in Shiny but aren't really used much in forms.  Let me know what you think.

I need to go to sleep now, I won't be active on this repo for a while FYI, just wanted to get a bunch of stuff implemented while I had the ideas fresh.  Oh and I downloaded your package to see the shiny app because it looks really nice in the pictures, but I couldn't figure out how to run it from your package, so I didn't see it.

**Comment by fozy81** [comment]
Good point about sliders. I suppose no harm in including them but not that useful.

The there's an example shiny app in the examples > shinyapp folder.

**Comment by daattali** [comment]
So I'm supposed to download that entire example folder to my computer and run it?  I assumed I'd be able to easily run it with a command

**Comment by fozy81** [comment]
here's the rough shinyapp, give you some ideas: https://opendata.shinyapps.io/rdatamill/

**Comment by daattali** [comment]
Oh that's very nice! Yeah looks like it's very geared towards experimental data collection

**Comment by fozy81** [comment]
Thanks, I wanted a demo for myself and hopefully inspire other people to work on experimental/laboratory data collection in shiny. But creating forms is at the heart of it and shinyforms is already better than my attempts! So maybe something like Radiant package - with different levels  from simple questionaire, to questionaire with validation, to complex science data collection...

**Comment by fozy81** [comment]
https://github.com/daattali/shinyforms/pull/18

Added rest of shiny input types, if this helps. plus a textarea/paragraph type. see what you think. 
I'm thinking a leaflet map input - for lat/lon etc could be interesting... e.g where's your favourite restaurant.
May want to hold off adding lots of input types until looking at how to configure all the input types/questions via shiny UI/Addins.

**Comment by daattali** [comment]
Great, thanks! I'll take a look eventually but not right now. Interesting idea re: maps.  How would the interaction be? "Here's a small map, click a location, we'll record the lot-lang"?   I've never seen it anywhere else, I'd be interested to see if others think that would actually be useful.

**Comment by fozy81** [comment]
I've added image question to my fork - see pull request above. Not really an input just a placeholder for images which can be placed between real input questions. I think it's a neat way to add images to a form. Could be handy for tests, machine learning, captchas...?
